css/layout.less

```diff
@@ -4,12 +4,21 @@
   height: 18px;
 }
 
-.navbar img.img-lg,
 .navbar .navbar-brand img {
   margin-top: -5px;
   height: 32px;
 }
 
+.navbar .cubing-icon {
+  font-size: 32px;
+  vertical-align: middle;
+}
+
+.navbar .navbar-brand .cubing-icon {
+  display: inline-block;
+  margin-top: -5px;
+}
+
```

Comment:
Since this is LESS - we could just nest the navbar and children classes. .navbar { .cubing-icon{} .navbar-brand .cubing-icon{} }. I definitely prefer this approach, since if we add anything else with .navbar as a parent, we don't have to duplicate that again (even though .navbar class probably won't change with bs3.x.y).